src/components/Auth/ResetPassword/ResetPassword.react.js

```diff
@@ -83,7 +83,7 @@ class ResetPassword extends Component {
       }.bind(this),
       error: function(errorThrown) {
         let state = this.state;
-        state.msg = 'Invalid token!';
+        state.msg = 'To reset password visit Homepage!';
```

Comment:
This error message would not make sense to be very honest. Actually, if you have an invalid token ( token which was not provided by the server ) then it is a case of malicious entry. In that case, "invalid user" or 'username not found' may be a better alternative.

Reply:
@AakashMallik Okay, So Should I change it to "Username not found"?

Reply:
@plxity IMO, yup. But ask @anshumanv before finalizing it.

Reply:
@AakashMallik Okay, sure!

Reply:
```suggestion
        state.msg = 'Invalid request!';
```
I think this would be better.